### PR TITLE
Sp3 has streetaddress

### PR DESCRIPTION
**Pull request by branduke**

### CODE REVIEW

File: pii_data.py [hunk 1]
```diff
@@ -8,7 +8,7 @@ class Pii(str):
     # https://regex101.com
     # https://www.w3schools.com/python/python_regex.asp
     def has_us_phone(self):
-        # Match a US phone number ddd-ddd-dddd ie 123-456-7890
+        # Match a US phone number ddd-ddd-dddd ie 123-456-7890s
```
Comment (claesmk) [inline pii_data.py:11]: Was the s here intended or just a type-o?  If it was an error you can remove it and push your branch again

**Review by claesmk — CHANGES_REQUESTED** [review]
I changed the base branch to NCATComp410:team1 instead of  NCATComp410:main

Please see the other comment

**Comment by claesmk** [comment]
Please also create an [issue](https://github.com/NCATComp410/comp410_spring_2022/issues)

File: pii_data.py (hunk 1 above)
Comment (branduke) [reply pii_data.py]: it was a typo, i fixed it.

**Review by claesmk — APPROVED** [review]
Fixed a few issues for you looks OK now

**Pull request merged**